handbook/engineering/distribution/recurring_processes.md

```diff
@@ -40,6 +40,8 @@ At the beginning of each cycle, [we determine what work we plan to do](#planning
 1. Revisit the goals. Are they still the right goals?
 1. If there are any remaining uncertainties following the meeting, the owners of each theme or task should follow up before the start of the cycle.
 
+**Backlog grooming:** The distribution team runs a backlog grooming session every two weeks to ensure that backlog is correctly priorotized for the upcoming sprint. We use the backlogs for tracking bugs, small features, and unplanned work. We don’t use the backlog for tracking work that is expressly planned in our roadmap. To add an issue, tag it team/distribution to notify the distribution team.
```

Comment:
```suggestion
**Backlog grooming:** The distribution team runs a backlog grooming session every two weeks to ensure that backlog is correctly prioritized for the upcoming sprint. We use the backlogs for tracking bugs, small features, and unplanned work. We don’t use the backlog for tracking work that is expressly planned in our roadmap. To add an issue, tag it `team/distribution` to notify the distribution team.
```

Reply:
Do we have tooling to notify on team tags? I remember seeing some other teams had it

Reply:
https://github.com/sourcegraph/sourcegraph/issues/17476#issuecomment-763812581